spring-batch/src/test/java/org/baeldung/conditionalflow/step/NumberInfoClassifierUnitTest.java

```diff
@@ -0,0 +1,17 @@
+package org.baeldung.conditionalflow.step;
+
+import org.baeldung.conditionalflow.model.NumberInfo;
+import org.baeldung.conditionalflow.step.NumberInfoClassifier;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class NumberInfoClassifierUnitTest {
+
+    @Test
+    void process() throws Exception {
```

Comment:
The same comment applies for test names.

Reply:
Done